docs/content/en/writing.md

```diff
@@ -367,7 +366,7 @@ Will be transformed into:
 }
 ```
 
-We internally add a `text` key with the markdown body that will be used for [searching](/fetching#searchfield-value) or [extending](http://localhost:3000/advanced#contentfilebeforeinsert) it.
```

Comment:
Why localhost here?

Reply:
Don't ask me. That's why I deleted it.

Reply:
The localhost was already there in the current docs. ;)

Reply:
Oh yeah you removed it, my bad!